### PR TITLE
T01 set up todo app

### PR DESCRIPTION
**Pull request by APaltrow**
Pull request on task #1 - T-01: Set ToDo app up

Check list:
1. Create the repository
2. Create root React project
3. Add and configure Redux to your project
4. Deploy to GitHub Pages

Notes: 
- Project should be written with React and Redux libraries.
- Design is up to you.
- Every task has to be in the separate branch.
- Every git branch should start with task number (T01_), and short description.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+node_modules/
+build/
+dist/
+
+assets/
+public/
+/**/*.d.ts
```
Comment (Yuliia143) [inline .eslintignore]: GitHub needs empty line at the end of each file you want to add

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+build
+dist
+node_modules
+assets
```
Comment (Yuliia143) [inline .prettierignore]: GitHub needs empty line at the end of each file you want to add

File: src/redux/reducers/todo.reducer.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/* TODO: refactor and split types separately */
```
Comment (Yuliia143) [inline src/redux/reducers/todo.reducer.ts]: Why is this file needed for this task?

File: .eslintignore (hunk 1 above)
Comment (APaltrow) [reply .eslintignore]: Added empty line for each file (also changed settings for my IDE)

File: .prettierignore (hunk 2 above)
Comment (APaltrow) [reply .prettierignore]: Added empty line for each file (also changed settings for my IDE)

File: src/redux/reducers/todo.reducer.ts (hunk 3 above)
Comment (APaltrow) [reply src/redux/reducers/todo.reducer.ts]: Removed irrelevant (for this task) TODO Reducer and reinitialized store with an empty object.

**Review by Yuliia143 — APPROVED** [review]
